doc/mgmt/SONiC_OC_FDB_get_support_HLD.md

```diff
@@ -0,0 +1,271 @@
+          |              +--rw interface-ref
+          |                 +--rw config
+          |                 |  +--rw interface?      -> /oc-if:interfaces/interface/name
+          |                 |  +--rw subinterface?   -> /oc-if:interfaces/interface[oc-if:name=current()/../interface]/subinterfaces/subinterface/index
+          |                 +--ro state
+          |                    +--ro interface?      -> /oc-if:interfaces/interface/name
+          |                    +--ro subinterface?   -> /oc-if:interfaces/interface[oc-if:name=current()/../interface]/subinterfaces/subinterface/index
+```
+### 3.6.2 CLI
+#### 3.6.2.1 Configuration Commands
+#### 3.6.2.2 Show Commands
+The following CLI commands dump the output of internal FDB entries from STATE_DB with various options (filters), for example, filter based on MAC address, entry type (static/dynamic), VLAN interface, physical/port-channel interfaces and MAC address table count ..etc.
+##### 3.6.2.2.1 show mac address-table
+Syntax
+
+show mac address-table [address ``<mac>`` | count | dynamic { address ``<mac>`` | vlan ``<id>`` | interface { Ethernet ``<id>`` | Port-channel ``<id>`` }} | interface { Ethernet ``<id>`` | Port-channel ``<id>`` } | static { address ``<mac>`` | vlan ``<id>`` | interface { Ethernet ``<id>`` | Port-channel ``<id>`` }} | vlan ``<id>`` }]
+
+Syntax Description:
+
+|    Keyword    | Description |
+|:-----------------:|:-----------:|
+| address `mac` | This option dumps the MACs matching the particular Mac
+| count | This option provides the no. of MAC entries present in the system
+| static | This option dumps the MAC entries matching the static entry type
+| dynamic | This option dumps the MAC entries matching the dynamic entry type
+| interface | This option dumps the MACs matching the particular physical/port-channel interface
+| vlan | This option dumps the MACs matching the particular VLAN
+
+
+Command Mode: User EXEC
+
+Example:
+````
+sonic# show mac address-table
+VlanId  Mac Address             Type            Interface
+1       90:b1:1c:f4:9d:83       dynamic         Ethernet1
+20      00:11:22:33:44:55       static          Ethernet0
+sonic#
+
+sonic# show mac address-table count
+MAC Entries for all vlans :
+Dynamic Address Count :                  1
+Static Address (User-defined) Count :    1
+Total MAC Addresses in Use:              2
+sonic#
+
+
+sonic# show mac address-table address 00:11:22:33:44:55
+VlanId  Mac Address             Type            Interface
+20      00:11:22:33:44:55       static          Ethernet1
+sonic#
+
+sonic# show mac address-table dynamic
+VlanId  Mac Address             Type            Interface
+1       90:b1:1c:f4:9d:83       dynamic         Ethernet1
+sonic#
+
+sonic# show mac address-table dynamic address 90:b1:1c:f4:9d:83
+VlanId  Mac Address             Type            Interface
+1       90:b1:1c:f4:9d:83       dynamic         Ethernet1
+
+sonic# show mac address-table vlan 1
```

Comment:
Why not `show mac address-table interface vlan 1` ?

Reply:
VLAN and mac-address are keys for FDB-Table so giving CLI as different options rather than inside interface options.